README.md

```diff
@@ -1,25 +1,32 @@
 # Th-Bingen
 IT-So17
-Names : 
+## Names
 - Florian Dickob 
 - Marco Christ 
 - Sven Jungblut 
 - Niklas Massmann
 
-Describing the Project :
+## Describing the Project
 - In version 1.0 we will set up the infrastucture for the whole project 
 - In version 2.0 we will create a code which is a simulation consisting of cars and an abstract, monolithic road network
 - In version 3.0 we will extend the simulation by randomizing the decision for whether each individual car can drive or gets delayed
 - In version 4.0 the simulation gets extended by adding more traffic participants
 
--Used Languages :
+## Used Languages
 - Code will be in english
 - Comments will be in english
 - Conversation will be in english
  
-Code of conduct :
+## Code of conduct
 [read Coc here!](https://github.com/SJ1337/Th-Bingen/blob/master/CODE_OF_CONDUCT.md)
 
-License :
+## License
 [read license here!](https://github.com/SJ1337/Th-Bingen/blob/master/LICENSE)
 
+## Launching the application
+After downloading the repository, open Windows Powershell and execute the following command after switching
+in the correct folder.
+
+**java -jar target/Th-Bingen-1.0-SNAPSHOT-jar-with-dependencies.jar**
```

Comment:
Good job, you got the file name right. 😃 I'd use fenced code blocks here instead of bold:

    ```
    java -jar target/Th-Bingen-1.0-SNAPSHOT-jar-with-dependencies.jar
    ```